app/views/layouts/koi/_navigation_header.html.erb

```diff
@@ -1,6 +1,10 @@
 <header>
   <h2 class="site-name"><%= URI.parse(root_url).host %></h2>
-  <h3 class="admin-name">Koi Admin</h3>
+  <% if local_assigns[:admin].present? %>
```

Comment:
There won't be a situation where there is no admin we require an admin to see this view.

This isn't the same as the design we discussed. Why did you decide to go with this approach rather than a footer for the nav bar? There's also no logout link here or profile picture.